pkg/backingstore/reconciler.go

```diff
@@ -1169,6 +1169,12 @@ func (r *Reconciler) needUpdate(pod *corev1.Pod) bool {
 		}
 	}
 
+	var pod_label = pod.Labels["backingstore"]
+	var bs_label = r.BackingStore.Labels["backingstore"]
```

Comment:
and we can check here if label is provided in backingstore then we will update the label in pv-pods.